src/dos/program_keyb.cpp

```diff
@@ -371,6 +385,9 @@ void KEYB::AddMessages()
 	        "    and code page.\n"
 	        "  - The [color=white]CPIFILE[reset], if specified, must contain the screen font for the given\n"
 	        "    [color=white]CODEPAGE[reset].\n"
+	        "  - The MS-DOS, DR-DOS, and Windows NT formats of the CPI files are supported\n"
```

Comment:
Drop the starting "The" from the start of the sentence.

Reply:
OK, changed.